examples/grpc_dynamic_routing/envoy_config.json

```diff
@@ -1040,32 +1035,14 @@
                               }
                             },
                             "serviceConfigId": "2019-12-16r0",
-                            "serviceName": "esp-grpc-echo-oxouww7xzq-uc.a.run.app"
+                            "serviceName": "examples-grpc-dynamic-routing-wd6ufmzfya-uc.a.run.app"
                           }
                         ]
                       }
                     },
                     {
                       "name": "envoy.filters.http.grpc_web"
                     },
-                    {
-                      "name": "envoy.filters.http.grpc_json_transcoder",
```

Comment:
why grpc_trancoder filter is removed

Reply:
Good catch, I skipped the large diff... Will fix.